File: src/test/java/org/ku/kc/logging/TestLoggingConfigurer.java
package org.ku.kc.logging;

import java.util.logging.*;

public class TestLoggingConfigurer {

  public TestLoggingConfigurer() {
    LogManager.getLogManager().reset();
    Logger.getLogger("").setLevel(Level.INFO);
    Logger.getLogger("").addHandler(new Handler() {
      @Override
      public void publish(LogRecord record) {
        System.out.println(new DefaultFormatter().format(record));
      }

      @Override
      public void flush() {
      }

      @Override
      public void close() throws SecurityException {
      }
    });
  }
}

File: src/test/scala/org/dauch/kc/logging/TestLoggingConfigurer.scala
package org.dauch.kc.logging

import java.util.logging._
import scala.util.chaining.scalaUtilChainingOps

class TestLoggingConfigurer {
  Logger.getLogger("").tap { rootLogger =>
    rootLogger.getHandlers.foreach(rootLogger.removeHandler)
    rootLogger.setLevel(Level.INFO)
    rootLogger.addHandler(TestLoggingHandler)
  }
}
